spec/factories/intakes.rb

```diff
@@ -321,7 +321,7 @@
 
   trait :with_faker_contact_info do
```

Comment:
Let's try stopping using Faker! Let's use incrementing always. For names, let's try to spell it out so it's "Cherry One" and "Cherry Two" etc.

Reply:
And always apostrophe in the last name why not.